i18n/i18n.go

```diff
@@ -0,0 +1,64 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
Can we move these other `.go` changes out of this PR? As it stands, the PR also includes all the i18n stuff so is hard to review in isolation.

Reply:
Fwiw, I agree

Reply:
Sorry the Go changes will disappear the moment I merge the first PR in the chain, and rebase this one on main. Sorry for the confusion. This is just a side effect of working on chain of PRs and Github does not allow a delta PR view (minus dependencies on previous PRs)

Reply:
Removed after rebase.